src/main.ts

```diff
@@ -830,6 +830,31 @@ export async function getRawLogs(executionGraphId: string, taskName: string, tas
   }
 }
 
+export function fileAddPadding(file: string): string {
+  // It's necesarry add the pading https://linuxhint.com/understand-base64-padding/
+  const runtimeParametersPadding = file.length % 4
+  switch (runtimeParametersPadding) {
+    case 2:
+      file += "=="
+      break
+    case 3:
+      file += "="
+      break
+    default:
+      break
+  }
+  return file
+}
+
+export function readParemetersFile(folderName, pipeline, runtimeParametersFile): string {
+  const runtimeParametersFilePath = path.join(folderName, runtimeParametersFile)
+  let runtimeParameters = Buffer.from(fs.readFileSync(runtimeParametersFilePath).toString().trim()).toString("base64")
+  runtimeParameters = fileAddPadding(runtimeParametersFilePath)
+  JSON.parse(fs.readFileSync(path.join(folderName, pipeline)).toString()).phases.verify.context.runtime_parameters =
```

Comment:
I really don't understand this part. It looks like you are reading the pipeline and then adding the runtimeParameters to it, but then on the next line we do returning the runtimeParameters. 

So I'm not really sure this line does anything at all.

Reply:
I updated this, tell me what is clearer.